homeassistant/components/nest/climate_sdm.py

```diff
@@ -303,6 +303,10 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         low_temp = kwargs.get(ATTR_TARGET_TEMP_LOW)
         high_temp = kwargs.get(ATTR_TARGET_TEMP_HIGH)
         temp = kwargs.get(ATTR_TEMPERATURE)
+        if ThermostatTemperatureSetpointTrait.NAME not in self._device.traits:
+            raise HomeAssistantError(
+                f"Error setting {self.entity_id} temperature to {kwargs}: Unable to find ThermostatTemperatureSetpointTrait in device traits {self._device.traits.keys()}"
```

Comment:
Please break long strings around 88 characters per line.

Reply:
Nevermind, seems I was viewing an old diff.

Reply:
Although I see the line is still sligthly longer (101) than 88 characters.

Reply:
https://github.com/home-assistant/core/pull/86989